manual/Integration/DBeaver.md

```diff
@@ -0,0 +1,56 @@
+# Integration with DBeaver
+
+[DBeaver](https://dbeaver.io/) is a SQL client software application and a database administration tool. For MySQL databases, it applies the JDBC application programming interface to interact with them via a JDBC driver. 
+
+Manticore allows you to use DBEaver for working with data stored in Manticore tables the same way as if it was stored in a MySQL database.
+
+## Settings to use
+
+To start working with Manticore in DBeaver, do the following steps:
+
+- Choose a `New database connection` option in DBeaver's UI
+- Choose `SQL` -> `MySQL` as DBeaver's database driver 
+- Set the `Server host` and `Port` options corresponding to the host and port of your Manticore instance  (keep `database` field empty) 
+- Set `root/<empty password>` as authentication credentials.
+
+
+## Functions avaliable
+
+Since Manticore does not support MySQL in full, only a part of DBeaver's functionality is available when working with Manticore.
+
+You will be able to: 
+
+- view, create, delete and rename tables
+- add and drop table columns 
+- insert, delete and update column data.
+
+You will not be able to:
+
+- use database integrity checks mechanisms ( `MyISAM` will be set as the only storage engine available)
+- use MySQL procedures, triggers, events, etc.
+- manage database users
+- set other database administration options
```

Comment:
```suggestion
## Functions available

Since Manticore does not fully support MySQL, only a part of DBeaver's functionality is available when working with Manticore.

You will be able to:
- View, create, delete, and rename tables
- Add and drop table columns
- Insert, delete, and update column data

You will not be able to:
- Use database integrity check mechanisms (`MyISAM` will be set as the only storage engine available)
- Use MySQL procedures, triggers, events, etc.
- Manage database users
- Set other database administration options
```